integration-tests/modules/__tests__/inventory/index.spec.ts

```diff
@@ -284,6 +284,97 @@ medusaIntegrationTestRunner({
         })
       })
 
+      describe("Update inventory levels", () => {
+        let locationId
+        let inventoryItemId
+        beforeEach(async () => {
+          const invItemReps = await api.post(
+            `/admin/inventory-items`,
+            { sku: "test-sku" },
+            adminHeaders
+          )
+
+          inventoryItemId = invItemReps.data.inventory_item.id
+
+          const sl = await appContainer
+            .resolve(ModuleRegistrationName.STOCK_LOCATION)
+            .create({ name: "test-location" })
+
+          locationId = sl.id
+
+          await api.post(
+            `/admin/inventory-items/${inventoryItemId}/location-levels`,
+            {
+              location_id: locationId,
+              stocked_quantity: 10,
+            },
+            adminHeaders
+          )
+        })
+
+        it("should update the stocked and incoming quantity for an inventory level", async () => {
+          const result = await api.post(
+            `/admin/inventory-items/${inventoryItemId}/location-levels/${locationId}`,
+            {
+              stocked_quantity: 15,
+              incoming_quantity: 5,
+            },
+            adminHeaders
+          )
+
+          expect(result.status).toEqual(200)
+          expect(result.data.inventory_level).toEqual(
```

Comment:
todo: as per the convention I believe it should return the inventory_item, @olivermrbl is that correct?